### ISSUE TITLE
Add checkbox for binary "Invalid Market" filter

### ISSUE THREAD

**Issue by pgebheim** [AugurProject/augur#2156]

**Comment by pgebheim** [comment]
Hey all, we missed this in creating this epic -- unless the design was already done and I'm not up to speed.

**Comment by Chwy5** [comment]
I don't know what this is.  What do you mean by this?

**Comment by matt-bullock** [comment]
![image](https://user-images.githubusercontent.com/39496225/57863122-c5c38d00-77f9-11e9-8b8e-54f8769bee3b.png)

Shoudn't need the design file as you already have all the components. Just waiting on the tooltip copy

**Comment by Chwy5** [comment]
wording....
Any Market where the current best bid/offer would profit as a result of a market resolving as invalid, will be filtered out if checked

**Comment by phoebemirman** [comment]
going to start the ui part of this

**Comment by Chwy5** [comment]
added - closing
